keystrokes/package.json

```diff
@@ -13,10 +13,12 @@
   "dependencies": {
     "firebase": "^5.2.0",
     "firebaseui": "^3.4.1",
+    "node-sass": "^4.12.0",
```

Comment:
So you can't just add node-sass unfortunately... 
I believe you'd have to eject from Create react app, although it might have changed since i last looked. 

You'll need to add it to your build pipeline in webpack. 
But it doesn't actually look like you're using sass at the moment, so probably can just take this out.

Reply:
Hmm interesting, that explains why it wasnt working when i just dropped it in.